gaugan/dataloader/commons.py

```diff
@@ -0,0 +1,6 @@
+import zipfile
+
+
+def extract_zip(archive_path: str, target_dir: str) -> None:
+    with zipfile.ZipFile(archive_path, "r") as zip_ref:
+        zip_ref.extractall(target_dir)
```

Comment:
Is it needed now since we are already downloading it and extracting it?

Reply:
No its not needed, gonna remove